std/cs/_std/sys/net/Socket.hx

```diff
@@ -48,6 +49,13 @@ class Socket {
 
 	public var custom:Dynamic;
 
+	/**
+		Creates a new unconnected socket.
+	**/
+	public function new() : Void {
```

Comment:
Duplicated field; seems like the merge messed up with `function init()` below now named `new()`